### PR TITLE
Add commands to display server version, health, and general info.

### PR DESCRIPTION
**Pull request by jjmason**

### CODE REVIEW

File: conjur.gemspec [hunk 1]
```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
 
 
   gem.add_dependency 'activesupport'
-  gem.add_dependency 'conjur-api', '~> 4.20', '>= 4.20.1'
+  gem.add_dependency 'conjur-api', '>= 4.20.3'
```
Comment (dividedmind) [inline conjur.gemspec]: Why remove the `'~> 4.20'`? This will make it try to use version 5 if available.

Comment (jjmason) [inline conjur.gemspec]: I wasn't sure how multiple version specifiers work.  Does it just check that they match in order?

Comment (kgilpin) [inline conjur.gemspec]: The third version is the "patch" version, which only indicates bug fixes. What we're trying to do here is make sure that the API has the features that the CLI needs, so only the first two numbers are important. Also, the API version can simply be 4.20.0, because we haven't released it to Rubygems yet (so we can keep adding stuff).

File: lib/conjur/version.rb [hunk 2]
```diff
@@ -19,6 +19,6 @@
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 module Conjur
-  VERSION = "4.29.0"
+  VERSION = "4.29.1"
```
Comment (kgilpin) [inline lib/conjur/version.rb]: When adding new features, the second number needs to be incremented. You can check https://rubygems.org/gems/conjur-cli to see what version is currently released. (It's 4.28.2). So we can keep adding features to 4.29.0. Incrementing the patch version would only be done if the gem was already released and we had to fix a bug.

**Comment by kgilpin** [comment]
I would be over the moon if you added to `https://github.com/conjurinc/cli-ruby/tree/master/acceptance-features` :-)

File: conjur.gemspec [hunk 3]
```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
 
 
   gem.add_dependency 'activesupport'
-  gem.add_dependency 'conjur-api', '~> 4.20', '>= 4.20.1'
+  gem.add_dependency 'conjur-api', '~> 4.21.0'
```
Comment (dividedmind) [inline conjur.gemspec]: This will make it not work with 4.22. You probably want `'~> 4.21'`.

File: conjur.gemspec [hunk 4]
```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |gem|
 
   gem.add_development_dependency 'rspec', '~> 3.0'
   gem.add_development_dependency 'simplecov'
-  gem.add_development_dependency 'aruba', '~> 0.12.0'
+  gem.add_development_dependency 'aruba', '~> 0.11.2'
```
Comment (dividedmind) [inline conjur.gemspec]: Why the change?

File: lib/conjur/command/server.rb [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+class Conjur::Command::Server < Conjur::Command
+  desc 'Show Conjur client and server versions'
+  command :version do |v|
+    v.action do |*_|
+      puts "Conjur client version #{Conjur::VERSION}"
```
Comment (dividedmind) [inline lib/conjur/command/server.rb]: I think this should show the CLI and API versions separately, or else indicate clearly which one this is.

Comment (jjmason) [inline lib/conjur/command/server.rb]: Good idea!
